pkg/virt-operator/kubevirt_test.go

```diff
@@ -515,26 +571,58 @@ var _ = Describe("KubeVirt Operator", func() {
 			}
 			kv.DeletionTimestamp = now()
 
+			// SCC will always be checked
+			shouldExpectSccGet(1, false)
```

Comment:
I forgot about this scc always performing an API call on every sync.  We need to see if there's a way to cache this scc information like we do with everything else. 

Thinking back on the initial operator PR, i remember it was this scc call that kept me from wanting us to address the ```if kv.Status.Phase == Deployed {return nil}``` conditional logic right away.

Reply:
good point, will try to get a SCC cache working

Reply:
done